From 4f62c986c8f9ea576ffd565f3d417c5bb5879e2a Mon Sep 17 00:00:00 2001
From: Peng Fan <peng.fan@nxp.com>
Date: Fri, 16 Nov 2018 16:10:06 +0800
Subject: [PATCH 5147/5242] MLK-20384 soc: imx: drop xenvm soc data

commit  bef231b948e40447a7a06d8ff856378446ed1b5b from
https://source.codeaurora.org/external/imx/linux-imx.git

Drop xenvm soc data, we have userspace tool stack generate correct model
compatile and this soc data will also trigger issue on i.MX8QXP.

So drop it.

Signed-off-by: Peng Fan <peng.fan@nxp.com>
Reviewed-by: Flynn xu <flynn.xu@nxp.com>
(cherry picked from commit 9be6db046ac435f2caab7ab3561ea1a66144cdda)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/soc/imx/soc-imx8.c |    2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/soc/imx/soc-imx8.c b/drivers/soc/imx/soc-imx8.c
index 5a5215d..000fa31 100644
--- a/drivers/soc/imx/soc-imx8.c
+++ b/drivers/soc/imx/soc-imx8.c
@@ -230,8 +230,6 @@ static u32 imx8mm_soc_revision(void)
 	{ .compatible = "fsl,imx8qxp", .data = &imx8qxp_soc_data, },
 	{ .compatible = "fsl,imx8mq", .data = &imx8mq_soc_data, },
 	{ .compatible = "fsl,imx8mm", .data = &imx8mm_soc_data, },
-	/* Fixme: this is a hack for big/little xen guest, b0 no need this */
-	{ .compatible = "xen,xenvm", .data = &imx8qm_soc_data, },
 	{ }
 };
 
-- 
1.7.9.5

